third_party/llm_engine/llm_engine.bzl

```diff
@@ -23,8 +23,8 @@ def llm_engine():
     llm_engine_repository(name="_llm_engine")
     new_git_repository(
         name = "llm_engine",
-        remote = "https://github.com/openvinotoolkit/openvino.genai",
-        commit = "b7e8993565358a9e2d8f69bc915bfbf8e5ea9ed4", # releases/2025/2 RC1
+        remote = "https://github.com/dkalinowski/openvino.genai",
+        commit = "127168c3f0e6865f90965ca4840c53160de9f002", # releases/2025/2 RC1 + ImageGen PR
```

Comment:
we can use RC2, image gen PR is already merged

Reply:
change to current latest releases/2025/2